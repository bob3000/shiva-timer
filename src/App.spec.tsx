import * as React from 'react';
import * as renderer from 'react-test-renderer';
import StandardTimerScreen from './App';

jest.useFakeTimers();

describe('App', () => {
  it('snapshot test', () => {
    const tree = renderer.create(<StandardTimerScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
