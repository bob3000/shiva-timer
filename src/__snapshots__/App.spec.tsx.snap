// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "height": "100%",
        "opacity": 0,
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "borderColor": "#000000",
          "borderRadius": 40,
          "borderWidth": 1,
          "marginTop": 100,
          "opacity": 0.8,
          "width": 280,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": undefined,
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontSize": 76,
                  },
                  Object {
                    "width": 95,
                  },
                  Object {
                    "textAlign": "right",
                  },
                ]
              }
            >
              10
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontSize": 76,
                  },
                ]
              }
            >
              :
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontSize": 76,
                  },
                  Object {
                    "width": 95,
                  },
                ]
              }
            >
              00
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderColor": "#000000",
          "borderRadius": 30,
          "borderWidth": 1,
          "justifyContent": "center",
          "marginTop": 50,
          "opacity": 0.8,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 16,
            "fontWeight": "bold",
            "marginTop": 10,
          }
        }
      >
        Warmup Time
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "padding": 15,
          }
        }
      >
        <RCTSlider
          disabled={false}
          enabled={true}
          maximumTrackTintColor="#000000"
          maximumValue={6}
          minimumTrackTintColor="#000000"
          minimumValue={0}
          onChange={[Function]}
          onResponderTerminationRequest={[Function]}
          onSlidingComplete={[Function]}
          onStartShouldSetResponder={[Function]}
          onValueChange={[Function]}
          step={0}
          style={
            Array [
              Object {
                "height": 40,
              },
              Object {
                "width": 200,
              },
            ]
          }
          thumbTintColor="#000000"
          value={2}
        />
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": undefined,
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontSize": 18,
                    "fontWeight": "bold",
                  },
                  Object {
                    "width": 25,
                  },
                  Object {
                    "textAlign": "right",
                  },
                ]
              }
            >
              00
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontSize": 18,
                    "fontWeight": "bold",
                  },
                ]
              }
            >
              :
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontSize": 18,
                    "fontWeight": "bold",
                  },
                  Object {
                    "width": 25,
                  },
                ]
              }
            >
              30
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginTop": 50,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Array [
              Object {
                "backgroundColor": "#FFFFFF",
                "opacity": 0.8,
              },
              Object {
                "alignItems": "center",
                "borderColor": "#000000",
                "borderRadius": 100,
                "borderWidth": 1,
                "height": 200,
                "justifyContent": "center",
                "opacity": 0.8,
                "width": 200,
              },
            ]
          }
        >
          <Image
            source={1}
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                Object {
                  "height": undefined,
                  "width": undefined,
                },
                undefined,
              ]
            }
          />
          <View
            style={
              Object {
                "borderRadius": 40,
                "opacity": 1,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000000",
                  "fontSize": 50,
                  "fontWeight": "bold",
                }
              }
            >
              
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "bottom": 0,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": 10,
          "position": "absolute",
          "width": "100%",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Array [
              Object {
                "backgroundColor": "#FFFFFF",
                "opacity": 0.8,
              },
              Object {
                "alignItems": "center",
                "borderColor": "#000000",
                "borderRadius": 100,
                "borderWidth": 1,
                "height": 60,
                "justifyContent": "center",
                "opacity": 0.8,
                "width": 60,
              },
            ]
          }
        >
          <Image
            source={1}
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                Object {
                  "height": undefined,
                  "width": undefined,
                },
                undefined,
              ]
            }
          />
          <View
            style={
              Object {
                "borderRadius": 40,
                "opacity": 1,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000000",
                  "fontSize": 15,
                  "fontWeight": "bold",
                }
              }
            >
              Reset
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "alignSelf": "flex-end",
            "backgroundColor": "#FFFFFF",
            "borderRadius": 20,
            "color": "#000000",
            "opacity": 0.8,
            "padding": 5,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <Text>
            about Shiva Timer
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
